extensions/markdown-reference/src/markdown-escape.ts

```diff
@@ -0,0 +1,21 @@
+const replacements: Array<Array<RegExp | string>> = [
+  [/\*/g, "\\*", "asterisks"],
+  [/#/g, "\\#", "number signs"],
+  [/\//g, "\\/", "slashes"],
+  [/\(/g, "\\(", "parentheses"],
+  [/\)/g, "\\)", "parentheses"],
+  [/\[/g, "\\[", "square brackets"],
+  [/\]/g, "\\]", "square brackets"],
```

Comment:
```suggestion
  [/]/g, "\\]", "square brackets"],
```
Removed unnecessary escape